pkgs/development/python-modules/bcolz/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, numpy, mock, setuptools_scm, isPy27, isPy35 }:
+
+buildPythonPackage rec {
+    pname = "bcolz";
+    version = "1.1.2";
+    name = "${pname}-${version}";
+
+    disabled = !(isPy35 || isPy27);
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "0ws0b9p1r4gcxic7fm8clm2q2ily3bd6i4giczw4s6gv5ilk7ds2";
+    };
+
+    doCheck = isPy27;
```

Comment:
why disable tests with other versions?

Reply:
I could only get the tests to pass on python2